cmd/watch.go

```diff
@@ -70,7 +70,7 @@ var watchCmd = &cobra.Command{
 		}
 		watchPath := util.ReadFilePath(u, runtime.GOOS)
 
-		err = component.WatchAndPush(client, componentName, applicationName, watchPath, stdout, ignores, delay)
+		err = component.WatchAndPush(client, componentName, applicationName, watchPath, stdout, ignores, delay, make(chan string))
```

Comment:
i've never seen this before... why are we passing in a channel?

Reply:
bcoz of https://github.com/redhat-developer/odo/pull/820#discussion_r228723172